cooper/optim/utils.py

```diff
@@ -13,8 +13,8 @@
 
 def create_optimizer_from_kwargs(
     primal_optimizers: OneOrSequence[torch.optim.Optimizer],
+    cmp: ConstrainedMinimizationProblem,
     dual_optimizers: Optional[OneOrSequence[torch.optim.Optimizer]] = None,
```

Comment:
I forget, did we agree to keep this function? It is only used for the tests.

Should we add a TODO?